drivers/mspi/Kconfig

```diff
@@ -8,6 +8,7 @@
 #
 menuconfig MSPI
 	bool "Multi-bit Serial Peripheral Interface (MSPI) bus drivers"
+	select EXPERIMENTAL
```

Comment:
This being a code change, maybe it should be in a different PR.

Reply:
I was going to leave it out. Is marking the version to be 0.1.0 enough to mark API as experimental?
I’m not sure what is the impact of adding select EXPERIMENTAL as none of mspi related code uses that anyway.

Reply:
AFAIK they are a not exactly the same thing. The 0.1.0 in the header just affects the docs, so it will be visible as experimental in the API overview page.
The EXPERIMENTAL kconfig would show in the build .config, and if the user enabled CONFIG_WARN_EXPERIMENTAL trigger a warning during the kconfig build step.

Reply:
I think it will be best if you could split the kconfig change in a separate PR and commit. That way we can merge the rest as is for 3.7.0.
The kconfig change we may merge also for 3.7.0, or instead it may go to the 3.7 branch and be in 3.7.1

Reply:
we should not need that, it experimental based on the API documentation below. We do not use this Kconfig for drivers that have a defined life cycle and a way to speficy stability.

Reply:
Thanks and removed.